src/main/java/roomescape/repository/reservation/ReservationJdbcRepository.java

```diff
@@ -0,0 +1,60 @@
+package roomescape.repository.reservation;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+
+@Repository
+public class ReservationJdbcRepository implements ReservationRepository {
+    private final JdbcTemplate jdbcTemplate;
+    private final SimpleJdbcInsert jdbcInsert;
+
+    public ReservationJdbcRepository(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.jdbcInsert = new SimpleJdbcInsert(jdbcTemplate)
+                .withTableName("reservation")
+                .usingGeneratedKeyColumns("id");
+    }
+
+    @Override
+    public Reservation add(Reservation reservation) {
+        Map<String, Object> params = new HashMap<>();
+        params.put("name", reservation.getName());
+        params.put("date", reservation.getDate());
+        params.put("time_id", reservation.getTime().getId());
```

Comment:
`Map.of` 를 활용해 간결하게 표현 가능합니다.

```suggestion
    Map<String, Object> params = Map.of(
        "name", reservation.getName(),
        "date", reservation.getDate(),
        "time_id", reservation.getTime().getId()
    );
```

Reply:
과거 레벨1 미션을 통해 Map.of()의 키-값 쌍 개수 제한(최대 10개)을 경험한 적이 있습니다.
현재 Reservation 클래스에서는 문제가 없지만, 향후 도메인 객체가 확장되어 필드가 10개를 초과할 경우를 대비하여 HashMap으로 명시적으로 초기화하고 put 메서드를 사용하는 방식을 선택했습니다.
이를 통해 잠재적인 사이드 이펙트를 최소화하고자 했습니다.